source/Willow/TableWebView.class.st

```diff
@@ -12,19 +12,32 @@ Class {
 		'interactionInterpreter',
 		'commandToRow',
 		'commandToTable',
-		'identifierAssigner'
+		'identifierAssigner',
+		'footer'
 	],
 	#category : #'Willow-WebViews'
 }
 
 { #category : #'instance creation' }
 TableWebView class >> definedBy: aColumnRendererCollection applying: aTableCommand headerRenderedBy: aHeader applyingToEachRow: aRowCommand [
 
+	^ self
```

Comment:
I wouldn't like to have an explosion of instance creation methods. Tables should always be built with all parameters, and the table builders in each supplier should know to use the null defaults if nothing is specified.

Reply:
I will go that route but we need to deprecate the previous ones first.. or release a major version ... Maybe we can do that as a two step process? I would like to review the methods in the builder also, because essentially are 3 aspects to consider now: headers, column definitions and footers.. By now I've just added the more generic method with footer support because if we start making each combination it will be an explosion. But now the builder it's a bit verbose  for the simple cases with footers :( . What do you think? Release it as is and open an issue to remove the extra instance creation methods and improving the table builder API, or wait for this changes to be made?